pages/platform/data-processing/36_HOWTO_submit-job-cli/guide.en-gb.md

```diff
@@ -0,0 +1,127 @@
+
+When it is created, upload your application code in your container. If you don't have any application code, you can still try the CLI with the examples files provided inside the ``testdata`` directory of the [GitHub project](https://github.com/ovh/data-processing-spark-submit){.external}.
+If you want to submit a python job, do not forget to upload your environment.yml file (see [How to generate environment file for Python jobs](../generate-environment){.external})
+
+>[!primary]
+>
+> If you only have your application code to upload, you can use the auto-upload feature of the CLI instead of uploading it manually (see **step 4: Submit a job - Optionally use auto-upload**).
+
+
+### Step 4: Submit a job
+
+Now everything is ready, let's submit a job !
+
+To launch a job with the Data Processing CLI, you have to run the executable file you previously built with your job configurations as parameters.
+
+Here is an example of command you could run to submit a SparkPi job in java/scala:
+
+```shell-session
+$ ./ovh-spark-submit --projectid yourProjectId --class org.apache.spark.examples.SparkPi --driver-cores 1 --driver-memory 4G --executor-cores 1 --executor-memory 4G --num-executors 1 swift://odp/spark-examples.jar 1000
+```
+
+In this example, the application code ``spark-examples.jar``is stored in the ``odp`` container on Object Storage, the Spark driver and its executor have 1 core and 4 gibibytes of memory. Once the Spark cluster deployed the script will run with ``1000`` as argument.
+
+>[!primary]
+>
+> Some of the parameters can be set as environment variables, such as ** --projectid**.
+
+The ovh-spark-submit CLI provides a part of the parameters of spark-submit to configure your job.
+If you want to know more about these parameters run:
+
+```shell-session
+$ ./ovh-spark-submit -h
+```
+
+While your job is running, you can watch logs in your terminal or access Spark UI through this URL: 
+
+``https://adc.{region}.dataconvergence.ovh.com/{your-job-id}/jobs/``
+
+At any time, you can stop your job by pressing ``Ctrl+C``. If you do so, the CLI will ask you to confirm that you want to cancel the before killing it.
+
+If you want to check your result after job has finished, you can download the logs of your job from your Object Storage (see [Checking a job's logs in the Data Processing manager's page](../check-logs)).
+
+#### Optionally use auto-upload
+If you want to save time when you often need to change your application code, the auto-upload feature of the CLI allows you to upload your code on Object Storage automatically.
+
+To enable it, you need to update the configuration.ini file to add the configurations needed for the protocol you want to use.
+
+For now, the only protocol supported is swift. Here is how you should update you configuration.ini file in order to upload your code with this protocol:
```

Comment:
OpenStack Swift

Reply:
Also, I see it here and in the line before so i probably missed it somewhere else. But could you put configuration.ini in a special font ?